fleet-packages-quickstart/README.md

```diff
@@ -0,0 +1,246 @@
+# Multi-cluster configuration with Fleet Packages
+
+[Fleet Packages] provides a way to deploy configuration into multiple clusters in a [fleet].
+
+## Prerequisistes
+
+1. [gcloud CLI], as this guide uses it to set up the necessary infrastructure and for using the Fleet Packages service.
+2. A GitHub account, as we will fetch the configuration from git. Authentication to git will be set up using a GitHub App, so you must have admin-level permissions on your GitHub repository.
+3. A Google Cloud project with billing and with GKE Enterprise enabled.
+
+
+## Configure the environment
+
+Create an environment variable in your shell that identifies the project you want to use:
+
+```
+PROJECT_ID=<MY_PROJECT>
+```
+
+Also set the current project in gcloud to the same project:
+
+```
+gcloud config set project $PROJECT_ID
+```
+
+If you want to use your current project, you can set the environment variable with:
+
+```
+PROJECT_ID=$(gcloud config get-value project)
+```
+
+## Install Config Sync across the fleet
+
+Fleet Packages is built on top of Config Sync, so we need to install Config Sync on all clusters.
+
+Create the apply-spec:
+
+```
+cat << EOF > apply-spec.yaml
+applySpecVersion: 1
+spec:
+  configSync:
+    enabled: true
+EOF
+```
+
+Enable the config-management feature on the fleet and set it up to install Config Sync on all new clusters:
+
+```
+gcloud beta container fleet config-management enable --fleet-default-member-config=apply-spec.yaml --project=$PROJECT_ID
+```
+
+
+## Set up a fleet
+
+We need a few GKE clusters registered to a fleet. We only create two clusters here to keep things simple.
+
+
+Create a cluster in us-west:
+```
+gcloud container clusters create fp-quickstart-cluster-west --project=$PROJECT_ID --zone=us-west2-a --workload-pool=$PROJECT_ID.svc.id.goog --enable-fleet --async
+```
+
+Create a cluster in us-east:
+```
+gcloud container clusters create fp-quickstart-cluster-east --project=$PROJECT_ID --zone=us-east1-c --workload-pool=$PROJECT_ID.svc.id.goog --enable-fleet --async
+```
+
+It will take a few minutes for the clusters to be ready. You can continue with the next steps and there is a later step to verify that they are ready.
+
+
+## Configure Cloud Build Repositories
+
+We have a sample package at https://github.com/GoogleCloudPlatform/anthos-config-management-samples/tree/main/fleet-packages-quickstart/config. This is a simple example that just sets up nginx. Fork this repo into your own GitHub account.
+
+Fleet Packages leverages [Cloud Build Repositories] (2nd gen) for connecting to git. Follow these steps to set it up:
+
+1. Enable the Cloud Build and Secret Manager APIs:
+```
+gcloud services enable cloudbuild.googleapis.com --project=$PROJECT_ID
+gcloud services enable secretmanager.googleapis.com --project=$PROJECT_ID
```

Comment:
It's not clear why Secret Manager API is needed. Would you add some clarification?

Reply:
Added some more details about why this is needed.